plugins/generator-1.15.2/forge-1.15.2/gamerule.definition.yaml

```diff
@@ -4,4 +4,6 @@ templates:
 
 localizationkeys:
   - key: gamerule.@registryname
+    mapto: displayName
```

Comment:
`displayName` is null on existing game rules. Please make an updater that sets displayName to the current description value to update existing game rules in workspaces.

Currently, NullPointerException is thrown when regenerating.

Reply:
Is an updater really needed? Gamerules were added in the first or second snapshot, not the last full update. Also, if you edit the Game rule mod element, you can fix the bug.

Reply:
Yes. Many people use snapshot.